pyproject.toml

```diff
@@ -13,11 +13,29 @@ packages = [
 ]
 
 classifiers = [
+  # How mature is this project? Common values are
+  #   3 - Alpha
+  #   4 - Beta
+  #   5 - Production/Stable
   'Development Status :: 4 - Beta',
+
+  # Indicate who your project is intended for
   'Intended Audience :: Developers',
+  'Intended Audience :: Science/Research',
+
+  'Topic :: Database :: Database Engines/Servers',
+
+  # Pick your license as you wish (should match "license" above)
   'License :: OSI Approved :: MIT License',
+
   'Natural Language :: English',
+
+  # Specify the Python versions you support here. In particular, ensure
+  # that you indicate whether you support Python 2, Python 3 or both.
+  'Programming Language :: Python :: 3',
   'Programming Language :: Python :: 3.6',
+
+  'Framework :: Pyramid',
```

Comment:
Not exactly accurate WRT this repo but not sure it matters.

Reply:
I agree. Doesn't need to be there.

I added Python 3.7 at the same time, since that's the number we define farther down.